### ISSUE TITLE
Long pressing @ - key feature

### ISSUE THREAD

**Issue by reinhard-weber** [florisboard/florisboard#224]
Hi and thanks for developing this cool keyboard app! I'd like to suggest a feature which I miss on every board: Long pressing the @ - key should insert a standard email string configurable on the settings page ...

**Comment by patrickgold** [comment]
Thanks for creating this issue! As already discussed, I really like this idea and will probably add it as a swipe up gesture on the "@" key to avoid the special menu conflict on the same key. Adding to the v0.5.0 milestone.

**Comment by Glitchy-Tozier** [comment]
@patrickgold What's a special menu conflict?

@ doesn't seem to have any other pop-ups.

**Comment by patrickgold** [comment]
@Glitchy-Tozier Yes, the "~left" key, in most layouts either the "@", "/" or "," key has the extended popups for Settings, emojis and one-handed mode on it. Long pressing conflicts because I don't know if you want to insert the email address or show the extended popup menu for these.

**Comment by Glitchy-Tozier** [comment]
Oh, that @-key makes more sense!
For a moment I thought you wanted to create a new gesture for the special-keyboard-@.

Another option could be this:
![emails](https://user-images.githubusercontent.com/59611881/105634486-70f56800-5e5e-11eb-99ce-3240056e488f.png)

1. I'd understand it if you don't want to add weird new pop-ups though.
2. ((look at how pretty the adapting g-board theme is!!))

**Comment by patrickgold** [comment]
This is indeed a nice idea, though it completely obliterates my current popup logic, as the `@gmail` label is wider than 1 key.

> ((look at how pretty the adapting g-board theme is!!))

:sad dark theme user noise:
_no but really the dark theme adaptive theme of Gmail is waaaaay to bright_. But I do agree it looks nice on light themed devices!

**Comment by Glitchy-Tozier** [comment]
> This is indeed a nice idea, though it completely obliterates my current popup logic, as the `@gmail` label is wider than 1 key.

That's what I was worried about  xD

> :sad dark theme user noise:
> _no but really the dark theme adaptive theme of Gmail is waaaaay to bright_. But I do agree it looks nice on light themed devices!


It's exactly the same as the day-Theme, from what I can tell 😅

**Comment by patrickgold** [comment]
> It's exactly the same as the day-Theme, from what I can tell 😅

Not if your system theme is dark...

**Comment by Glitchy-Tozier** [comment]
Oh, I just tried it with FlorisBoard's "always dark" setting and by forcing Gmail into dark mode.

Yep, agreed, the red is too bright.

**Comment by asandikci** [comment]
Cool idea! Any update on this feature?
